scripts/src/java/org/oppia/android/scripts/common/ScriptExemptions.kt

```diff
@@ -0,0 +1,632 @@
+package org.oppia.android.scripts.common
+
+/**
+ * List of files which should be exempted for the test file check.
+ *
+ * NOTE TO DEVELOPERS: New files should rarely be added here. In general, all new new files should
+ * have tests. If you think there's a valid exception, please specifically call this out in your PR
+ * description.
+ */
+object ScriptExemptions {
+  /** List of files which should be exempted for the test file check. */
+  val TEST_FILE_CHECK_EXEMPTIONS_LIST = listOf(
+    "TestCoroutineDispatcherTestBase.kt",
```

Comment:
Hmm. Should these have relative paths to the project root? I'm thinking about the case where two files happen to have the same name (it's technically possible).

Also, once we figure out the final format (i.e. relative paths) & layout (i.e. textproto per other comment), please lexicographically sort these so that they're easier to find (the paths will really help with that, I think).

Reply:
Revamped to relative path approach, is also lexicographically sorted.